### DISCUSSION TITLE
Best BiBTex Software for Mac

### DISCUSSION THREAD

**Discussion by josefhajda** [retorquere/zotero-better-bibtex#2819] [Q&A]
Hi,
just installed Zotero and this awesome plugin. Autoexport is activated so this is all fine but I need to restart Bibdesk every time a new entry is added. They do not appear automatically in bibdesk. Can anyone recommend me a Bibtex software for Mac where the new entries which are wrote to the file automatically show up?
Thanks!

**Comment by retorquere** [comment]
Asking the BibDesk author whether BibDesk can be made to detect on-disk changes will get you the best information on whether this is possible, and if so, how.

Just as a matter of curiosity, why do you use this combination?

**Comment by josefhajda** [comment]
Hello @retorquere 
alright. I will try ask them.... and about the combination... no idea. I once bought a license for Texifier which is a nice latex editor for Mac and I use it. I think BibDesk just came with the package. Never thought about it. but now when preparing for a master thesis it seems like Zotero would be a more capable management software and I can not directly cite in latex from Zotero (I think).

I hope This was, what you meant with your question.... do you have any other ideas or suggestions for a good combination?

**Reply by retorquere** [reply]
> I can not directly cite in latex from Zotero (I think)

I'm not sure what you mean. Is there some kind of integration between bibdesk and texifier? What do you actually do in bibdesk if you also use zotero?

**Reply by josefhajda** [reply]
Hello @retorquere 
sorry for expressing myself so strange. 
at the moment, I open my document (I have a template I use) and have a folder with everything in it, including the bib file. I use bibdesk to manage my cites and it also creates the names which I use within Texifier to cite certain documents. 

I am not sure, if there is a direct connection between Zotero and Latex, so I need to create the bib file. But I just realize that probably I don't need BibDesk, when Zotero manages the bib file properly... so perhaps this is just me being stupid.... I will check.

**Reply by josefhajda** [reply]
Alright.... I am stupid..... 
Zotero creates a Bibfile with cite keys and I can just use these from texifier. I don't need a bibtex software because the file is created properly and a few seconds after adding a DOI to Zotero I can cite it.... so the problem I thought I have is not really a problem...

**Reply by retorquere** [reply]
That's what I was trying to understand 👍
